### PR TITLE
feat(content-answers): Content Answers Modal Error

### PR DESCRIPTION
**Pull request by greg-in-a-box**
image

### CODE REVIEW

File: src/features/content-answers/ContentAnswersModalError.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@import '../../styles/variables';
+
+.bdl-ContentAnswersModalError {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
```
Comment (benjamin-shen) [inline src/features/content-answers/ContentAnswersModalError.scss:8]: is this necessary?

**Review by benjamin-shen — DISMISSED** [review]

File: src/features/content-answers/ContentAnswersModalError.scss (hunk 1 above)
Comment (greg-in-a-box) [reply src/features/content-answers/ContentAnswersModalError.scss:8]: yea, without it, it wasnt centering in the modal, content within the modal isnt center by default

Comment (benjamin-shen) [reply src/features/content-answers/ContentAnswersModalError.scss:8]: the `width: 100%`?

Comment (greg-in-a-box) [reply src/features/content-answers/ContentAnswersModalError.scss:8]: oh you meant the width? yea we can remove that

Comment (greg-in-a-box) [reply src/features/content-answers/ContentAnswersModalError.scss]: removed

**Review by megansmith-box — APPROVED** [review]
lgtm

**Pull request merged**